mappings/net/minecraft/tag/RequiredTagList.mapping

```diff
@@ -25,6 +27,7 @@ CLASS net/minecraft/class_5120 net/minecraft/tag/RequiredTagList
 	METHOD method_33147 addToManager (Lnet/minecraft/class_5415$class_5749;)V
 		ARG 1 manager
 	METHOD method_33148 getRegistryKey ()Lnet/minecraft/class_5321;
+		COMMENT {@return the registry key representing the type of object in this list}
```

Comment:
```suggestion
		COMMENT @return the registry key representing the type of object in this list
```

Reply:
Inline `@return` has special behavior, it generates the brief text too.

Reply:
alright, didn't know that

Reply:
Just fyi, 

> [As an inline tag, provides content for the first sentence of a method's description, and a "Returns" section, as if `@return` description were also present. In the default English locale, the first sentence is `Returns` *description* `.`](https://docs.oracle.com/en/java/javase/16/docs/specs/javadoc/doc-comment-spec.html#return)